servicecontrol/installation.md

```diff
@@ -86,4 +86,39 @@ If this is a new installation of ServiceControl click on the `Add New Instance`
 
 When adding the first instance of the ServiceControl service the default service name is "Particular.ServiceControl". It is possible choose to change this name to a custom service name. In doing so this is also changing the queue name associated with this instance of ServiceControl.
 
-The ServiceControl queue name is required for configuring the endpoint plugins. See [Install Heartbeats plugin](/monitoring/heartbeats/install-plugin.md) and [Install Custom Checks plugin](/monitoring/custom-checks/install-plugin.md) for more information.
\ No newline at end of file
+The ServiceControl queue name is required for configuring the endpoint plugins. See [Install Heartbeats plugin](/monitoring/heartbeats/install-plugin.md) and [Install Custom Checks plugin](/monitoring/custom-checks/install-plugin.md) for more information.
+
+
+## Removing ServiceControl
+
+To perform a clean uninstallation of ServiceControl from a machine:
+
+- Remove each ServiceControl instance using ServiceControl Management (or Powershell)
+- Uninstall ServiceControl
+
+### Remove ServiceControl instance
+
+To remove a ServiceControl instance, click the Advanced Options button and then select Remove. Optionally select to remove the database and logs directories and then confirm. This will stop the running instance (if it is running) and remove all files related to the instance from the local file system.
+
+#### Remaining artifacts
+
+Even after a ServiceControl instance has been removed, it can leave behind some artifacts. The ServiceControl instance can leave queues in the configured transport. The queue names will depend on the configuration of the ServiceControl instance.
```

Comment:
Maybe instead of `can leave behind` `leaves behind`? I am not a native English speaker and `can leave` implies to me that it sometimes does it but sometimes not.